src/components/forging/forgingTitle.js

```diff
@@ -21,7 +22,7 @@ class ForgingTitle extends React.Component {
               {this.props.account.delegate.username}
             </h2>
             <span>
-              <LiskAmount val={this.props.statistics.total} roundTo={2} /> LSK Earned
+              <LiskAmount val={this.props.statistics.total} roundTo={2} /> LSK {this.props.t('Earned')}
```

Comment:
`LSK` should be part of the translated string `{this.props.t('LSK Earned')` because other languages can have a different word order.